lib/elixir/lib/system.ex

```diff
@@ -363,6 +363,22 @@ defmodule System do
   end
 
   @doc """
+  Environment variable value.
+
+  Returns the value of the environment variable
+  `varname` as a binary or raises `RuntimeError`
+  if the environment variable is undefined.
+  """
+  @spec get_env!(binary) :: binary
+  def get_env!(varname) when is_binary(varname) do
+    case get_env(varname) do
+      nil ->
+        raise RuntimeError, message: "environment variable #{varname} is undefined"
```

Comment:
I'd probably do `s/environment/system environment/` as there is the Application environment and that could be confusing.